### ISSUE TITLE
Since Version 11.3.0: incorrect display of multiple series when using zones

### ISSUE THREAD

**Issue by Glopter** [highcharts/highcharts#20539]
#### Expected behaviour

All series in a diagram with zones should be displayed. Even if the values of series and zones are far apart.
(this worked until version 11.2.1)

#### Actual behaviour

Some series (the blue one in the example) in a diagram with zones are hidden or not visible.

#### Live demo with steps to reproduce
https://jsfiddle.net/8j3us9v5/

#### Product version
Highcharts 11.3.0

#### Affected browser(s)
Chrome, Edge. Firefox

**Comment by hubertkozik** [comment]
Hi @Glopter!

I cannot find anything not working in the demo, which you've provided. The first series is not visible, because you've set a white color to a zone from -1000 to 1400, and between those values, the line color is white, which matches the plot background color. You can check it by changing from white to green color, like in this demo: https://jsfiddle.net/BlackLabel/p50cfnh6/ or changing the plot background color: https://jsfiddle.net/BlackLabel/f4pawgbs/

Please, confirm if I understand you correctly and if you don't see any bug here.

**Comment by Glopter** [comment]
Hello @hubertkozik,
you are right. This example only has an easy issue with the colors.  
I had not provided the original series and configuration and thought I had successfully reproduced the phenomenon with these simple series. I need to make a new example.

**Comment by Glopter** [comment]
Hello @hubertkozik ,

here is the new example. It differs from the original in that the original in Angular receives the zones "subsequently" via ViewChild. And that seems to be the problem. I'm trying to mock this with the "Add Zones" button. Here are the steps to reproduce it:

GIVEN I opened https://codepen.io/SEW-Andre/pen/YzgrPxw
WHEN I click on the blue series legend
THEN the blue series is hidden ✔

WHEN I click on the blue series legend
THEN the blue series is displayed ✔

WHEN i click on the "Add Zones" button
THEN there is no change ❓

WHEN I click on the blue series legend
THEN the blue series is hidden ✔
AND the red series is shown with zone colors ✔

WHEN I click on the blue series legend
THEN the blue series remains hidden ❌

In a nutshell: Once the zones are defined by clicking "Add Zones", it is no longer possible to display both series at the same time.

I can work around the problem in my application by delivering the zones directly with the configuration. 
So just take note that since version 10.3.0 the zones don't seem to work like they used to. There's no rush on my part to fix it.

**Comment by hubertkozik** [comment]
Hi @Glopter!

To update the zones on an existing chart you should update the series by using `series.update` method, like in the demo below. You shouldn't change the chart object statically, because it doesn't fire the redraw function, which is necessary in Highcharts to apply changes.

**Live demo:** https://codepen.io/hubertkozik/pen/yLwEOwP
**API Reference:** https://api.highcharts.com/class-reference/Highcharts.Series#update

**Comment by Glopter** [comment]
Thank you @hubertkozik,
I got that that. It solves the "THEN there is no change ❓" issue but not my actual problem that after adding the zones both series no longer appear at the same time. It is still present in the live demo you provided.

**Comment by hubertkozik** [comment]
Thank you for a detailed explanation. I overlooked that previously and now I understand everything correctly. This issue is a duplicate of https://github.com/highcharts/highcharts/issues/20569 and there is the same workaround, which fixes your issue. Also, this bug is in progress, it will be fixed in the next version of Highcharts.

If you want to use the workaround in your project, simply copy-paste the IIFE function.
**Live demo with a workaround:** https://codepen.io/hubertkozik/pen/XWGYORg
